libs/core/config_registry/tests/unit/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@
 
 set(tests config_registry)
 
+set(config_registry_FLAGS COMPILE_FLAGS -DHPX_MODULE_STATIC_LINKING=1)
```

Comment:
Hmm, what prompted this change?

Reply:
MSVC generates a warning if an executable is linked against symbols from a static library that are marked as imported. This change fixes the warning.

Reply:
I see, thanks! Let's make it `-DHPX_MODULE_STATIC_LINKING` maybe? We don't set a value elsewhere.